### ISSUE TITLE
Flat styles: Unsupported operator in

### ISSUE THREAD

**Issue by StefanBrand** [openlayers/openlayers#15635]
**Is your feature request related to a problem? Please describe.**
I tried to apply the `in` flat style expression. I get an error message: `Unsupported operator in`.

**Describe the solution you'd like**
The `in` expression should be functional, as advertised in the docs: https://openlayers.org/en/latest/apidoc/module-ol_style_expressions.html 

**Example**
```json
      "style": [
        {
          "filter": [
            "in",
            ["get", "ori_crop_id"],
            ["literal", ["701", "721", "707", "716", "723", "704"]]
          ],
          "style": {
            "fill-color": "#4DAF4A"
          }
        }
      ]
```

**Workaround**
```json
      "style": [
        {
          "filter": [
            "any",
            ["==", ["get", "ori_crop_id"], "701"],
            ["==", ["get", "ori_crop_id"], "704"],
            ["==", ["get", "ori_crop_id"], "707"],
            ["==", ["get", "ori_crop_id"], "716"],
            ["==", ["get", "ori_crop_id"], "721"],
            ["==", ["get", "ori_crop_id"], "723"]
          ],
          "style": {
            "fill-color": "#4DAF4A"
          }
        }
      ]
```

PS.: Related to #15372

**Comment by fredj** [comment]
Hi @StefanBrand, the `in` operator exists for canvas but it was implemented after the 9.0.0 release

**Comment by StefanBrand** [comment]
Oh, thank you for the heads-up, I was not aware. :blush: Closing this again then. :) :handshake:
